<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1343</width>
    <height>1327</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>CV_Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonStartCapture">
        <property name="text">
         <string>StartCapture</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonReset">
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCrntCams">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>ShowCams</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="condComboBox">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>1~9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10~18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19~27</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28~36</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>37~45</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>46~54</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>55~63</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>64~72</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>73~81</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>82~90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>91~99</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100~108</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>109~117</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>118~126</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>128~135</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>138~144</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>145~153</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>154~162</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>163~171</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>172~180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>181~189</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>190~198</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>199~207</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>208~216</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>217~225</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>226~234</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>235~243</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>244~252</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>253~261</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>262~270</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOpenAIProject">
        <property name="text">
         <string>OpenAIProj</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOpenImgList">
        <property name="text">
         <string>OpenImgList</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonProcess">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Process</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxSaving">
        <property name="text">
         <string>SaveAll</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="PictureView" name="imagebox0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
        <property name="rubberBandSelectionMode">
         <enum>Qt::ContainsItemBoundingRect</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1000</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="PictureView" name="imagebox1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
        <property name="rubberBandSelectionMode">
         <enum>Qt::ContainsItemBoundingRect</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1000</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="PictureView" name="imagebox2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
        <property name="rubberBandSelectionMode">
         <enum>Qt::ContainsItemBoundingRect</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1000</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="PictureView" name="imagebox3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
        <property name="rubberBandSelectionMode">
         <enum>Qt::ContainsItemBoundingRect</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1000</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="PictureView" name="imagebox4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
        <property name="rubberBandSelectionMode">
         <enum>Qt::ContainsItemBoundingRect</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1000</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="PictureView" name="imagebox5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
        <property name="rubberBandSelectionMode">
         <enum>Qt::ContainsItemBoundingRect</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1000</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="PictureView" name="imagebox6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
        <property name="rubberBandSelectionMode">
         <enum>Qt::ContainsItemBoundingRect</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1000</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="PictureView" name="imagebox7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
        <property name="rubberBandSelectionMode">
         <enum>Qt::ContainsItemBoundingRect</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1000</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="PictureView" name="imagebox8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
        <property name="rubberBandSelectionMode">
         <enum>Qt::ContainsItemBoundingRect</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1000</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1343</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pictureview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
